tests/network/libs/bgp.py

```diff
@@ -217,6 +222,7 @@ def deploy_external_frr_pod(
         containers=containers,
         volumes=volumes,
         client=client,
+        share_process_namespace=True,
```

Comment:
Why is this required? It is an extreme and should be rarely used.

Reply:
IMO you should be able to exec into the correct container. If the current helpers cannot support it, we should add support for it.
Because it is a test, I guess the security concern is not high, but it is a but ugly.

We can take it in a follow up if you see it is not possible to exec into the right container.